packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -243,6 +243,196 @@ class _LinearProgressIndicatorPainter extends CustomPainter {
 /// The minimum height of the indicator can be specified using [minHeight].
 /// The indicator can be made taller by wrapping the widget with a [SizedBox].
 ///
+/// {@tool sample --template=stateless_widget_scaffold}
+///
+/// This example shows a [LinearProgressIndicator] with a fixed value.
+///
+/// ![https://flutter.github.io/assets-for-api-docs/assets/material/linear_progress_indicator_fixed_value.png]
```

Comment:
To add an image, there needs to be a Flutter sample that in this repo: https://github.com/flutter/assets-for-api-docs/tree/master/packages/diagrams/lib/src

Reply:
Since these samples are using DartPad, it's probably okay to remove these images, since viewers can see the the DartPad sample for a visual.

Reply:
@johnpryan alright, I added some images for samples. The ones which do not have any animations are samples. So I added images for them.

Should I add a file to generate the assets? Or should I remove the images altogether? 

Thanks

Reply:
I think we should take them out in this PR. You are free to follow up with a PR to https://github.com/flutter/assets-for-api-docs and then add them here.